serenity/metrics_helper.hpp

```diff
@@ -93,6 +93,10 @@ inline Try<double_t> CountIpc(const ResourceUsage_Executor& previous,
                     previous.statistics().perf().cycles();
   double_t instructionsPerCycle = instructions / cycles;
```

Comment:
the instructions (cycles and all perf counters) represent the instructions, etc that has been collected in a window of time. So what we need to do instead (which is simpler is):

IPC: current.statistics().perf().instructions() / current.statistics().perf().cycles()
IPS: current.statistics().perf().instructions() / current.statistics().perf().duration()